deploy/agent/kindling-collector-config.yml

```diff
@@ -122,12 +122,10 @@ exporters:
     metric_aggregation_map:
       kindling_entity_request_total: counter
       kindling_entity_request_duration_nanoseconds_total: counter
-      kindling_entity_request_average_duration_nanoseconds: histogram
       kindling_entity_request_send_bytes_total: counter
       kindling_entity_request_receive_bytes_total: counter
       kindling_topology_request_total: counter
       kindling_topology_request_duration_nanoseconds_total: counter
-      kindling_topology_request_average_duration_nanoseconds: histogram
```

Comment:
Please also remove the average time from aggregate_kind_map, otherwise, the collector may print logs of 'Undefined metricKind' in the log

Reply:
I have lowered the log level to `debug` to avoid this problem. It will be too inconvenient for users to add them back if we remove the average time from aggregate_kind_map, so I prefer to remaining it.

Reply:
We can also only remove the average time from aggregate_kind_map and keep them in the exporter.metric_aggregation_map.

Reply:
Hmm, it is an option indeed. But we can't control which histogram aggregated in that way.